#goal
sensor_msgs/PointCloud2 cloud_input
geometry_msgs/Point ignore_origin
std_msgs/Float32 ignore_start_angle
std_msgs/Float32 ignore_end_angle
---
#result
sensor_msgs/PointCloud2 shelf_points
sensor_msgs/PointCloud2[] shelves
geometry_msgs/Quaternion[] shelf_coefs
visualization_msgs/Marker[] shelf_bounding_boxes
visualization_msgs/Marker shelf_bounding_box
sensor_msgs/PointCloud2[] objects
visualization_msgs/Marker[] object_bounding_boxes
---
#feedback
